### DISCUSSION TITLE
Better Integration with gsap Scroll Trigger

### DISCUSSION THREAD

**Discussion by blitzve0** [idiotWu/smooth-scrollbar#396] [Ideas]
It is very apparent that people using this library are only the most advanced users who definitely are using scroll trigger animations in their projects, well atleast speaking for myself, and currently gsap Integration has to be done with plugin system and it's not a joyful experience if we want to use animations on multiple pages as we have to declare smooth-scrollbar on all of those pages.

**Comment by dolphin-wood** [comment]
It would be a great help if you can provide your plugin here, as I know little about GASP.

**Reply by blitzve0** [reply]
😂😂 I'm just a beginner i don't know how to do that sorry

**Comment by ehsan-shv** [comment]
@blitzve0 Hey! I've just published [react-nice-scroll](https://github.com/ehsan-shv/react-nice-scroll) package. this is about your idea. You can check the code or just use it to solve your issue.

**Reply by blitzve0** [reply]
Demn Thanks Bro I'm gonna use this for my website
